typescript/utils/src/messages.ts

```diff
@@ -67,3 +74,19 @@ export function parseMessage(message: string): ParsedMessage {
   const body = utils.hexlify(buf.slice(BODY_OFFSET));
   return { version, nonce, origin, sender, destination, recipient, body };
 }
+
+export function parseTokenMessage(messageBody: string): ParsedTokenMessage {
```

Comment:
Just curious, does this work for all protocols?

Reply:
yes!